src/main/java/org/commcare/formplayer/aspects/MetricsAspect.java

```diff
@@ -53,6 +56,15 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
             domain = bean.getDomain();
         }
 
+        if (args != null && args.length > 0 && args[0] instanceof SessionNavigationBean) {
+            SessionNavigationBean bean = (SessionNavigationBean) args[0];
+            String requestInitiatedByTag = bean.getRequestInitiatedByTag();
+            if (requestInitiatedByTag != null) {
+                System.out.println("requestedInitiateydby is " + requestInitiatedByTag);
```

Comment:
do we need this for prod ?

Reply:
no, deleted here [5feea3c](https://github.com/dimagi/formplayer/pull/1551/commits/5feea3c80f5ed361b9add353d38d52757afd994a)